ReadThread.py

```diff
@@ -48,19 +48,21 @@ def run(self):
 
             data = self.dataQueue.get(block=True, timeout=None)  # Block until something new
             self.dataQueue.task_done()
-
-            byteList = [[x] for x in data]  # sad that we still need this
+            print(data)
```

Comment:
Is this necessary? I feel like the console gets more than enough usage already.

Reply:
Nope. To be removed.